src/en/manganelo/src/eu/kanade/tachiyomi/extension/en/manganelo/Manganato.kt

```diff
@@ -2,20 +2,92 @@ package eu.kanade.tachiyomi.extension.en.manganelo
 
 import eu.kanade.tachiyomi.multisrc.mangabox.MangaBox
 import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import okhttp3.Headers
+import okhttp3.HttpUrl.Companion.toHttpUrl
 import okhttp3.Request
-import java.text.SimpleDateFormat
-import java.util.Locale
 
-class Manganato : MangaBox("Manganato", "https://manganato.com", "en", SimpleDateFormat("MMM dd,yy", Locale.ENGLISH)) {
+class Manganato : MangaBox("Manganato", "https://www.natomanga.com", "en") {
     override val id: Long = 1024627298672457456
 
-    // Nelo's date format is part of the base class
-    override fun popularMangaRequest(page: Int): Request = GET("$baseUrl/genre-all/$page?type=topview", headers)
-    override fun popularMangaSelector() = "div.content-genres-item"
-    override val latestUrlPath = "genre-all/"
+    override fun headersBuilder(): Headers.Builder = super.headersBuilder().set("Referer", "https://www.natomanga.com/") // for covers
```

Comment:
```suggestion
    override fun headersBuilder(): Headers.Builder = super.headersBuilder().set("Referer", "$baseUrl/") // for covers
```

... and all other comments for Mangakakalot applies for Manganato too.